# Практическая работа

**Цели работы**

Научиться строить точный доверительный интервал для среднего.

**Что нужно сделать**

Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением $\sigma=5$.

Найдите доверительный интервал для оценки математического ожидания $\mu$ с надёжностью 0.95, если выборочная средняя $\bar x = 24.15$, а объем выборки $n=100$.

Для этого выполните следующее:
1. Выберите подходящую формулу доверительного интервала.
2. Примените формулу для доверительного интервала.
3. Получите конкретный доверительный интервал для среднего.

Результатом будут границы доверительного интервала.

**Рекомендации по выполнению**

- Выполняйте задания в приведённой последовательности.
- Пользуйтесь материалами, о которых говорилось в видео.
- Используйте подсказки и справочные значения.

**Теоретическая подсказка:**

- Асимптотический доверительный интервал для среднего:

$$P \left(\bar x-z_{1-\frac{\alpha}{2}}\cdot \frac{\hat\sigma}{\sqrt{n}} \leq {\mu} \leq \bar x+z_{1-\frac{\alpha}{2}}\cdot \frac{\hat\sigma}{\sqrt{n}} \right) = 1-\alpha$$

- Точный доверительный интервал для среднего, дисперсия известна:
$$P \left(\bar x-z_{1-\frac{\alpha}{2}}\cdot \frac{\sigma}{\sqrt{n}} \leq {\mu} \leq \bar x+z_{1-\frac{\alpha}{2}}\cdot \frac{\sigma}{\sqrt{n}} \right) = 1-\alpha$$

- Точный доверительный интервал для среднего, дисперсия неизвестна:
$$P \left(\bar x-t_{1-\frac{\alpha}{2}}\cdot \frac{s}{\sqrt{n}} \leq {\mu} \leq \bar x+t_{1-\frac{\alpha}{2}}\cdot \frac{s}{\sqrt{n}} \right) = 1-\alpha$$

**Справочные значения:**
$$ \alpha = 5\%, z_{1-\frac{\alpha}{2}} = 1.96$$
$$ \alpha = 5\%, t_{1-\frac{\alpha}{2}} = 2.26$$

**Что оценивается**

- Выбрана верная формула для доверительного интервала.
- Получены верные границы для доверительного интервала.

**Как отправить работу на проверку**

Отправьте решение на проверку в конце модуля. К решению прикрепите:

- расчёты на бумаге / в документе;
- (необязательно) если пишете на latex, jupyter notebook с решением.

# Решение

Т.к. объем выборки сравнительно не велик, использовать будем точный доверительный интервал при условии, что известна дисперсия.

Тогда для $\alpha$ = 5% (надежность 0.95) формула определения доверительного интервала будет выглядеть:

$$P \left(\bar x-z_{1-\frac{\alpha}{2}}\cdot \frac{\sigma}{\sqrt{n}} \leq {\mu} \leq \bar x+z_{1-\frac{\alpha}{2}}\cdot \frac{\sigma}{\sqrt{n}} \right) = 1-\alpha$$

С учетом:

$$ \alpha = 5\%, z_{1-\frac{\alpha}{2}} = 1.96$$

Получим:

$$P \left(\bar x-1.96\cdot \frac{\sigma}{\sqrt{n}} \leq {\mu} \leq \bar x+1.96\cdot \frac{\sigma}{\sqrt{n}} \right) = 1-\alpha$$

Все значения нам известны, подставим их в формулу:

$$P \left(24.15-1.96\cdot \frac{5}{\sqrt{100}} \leq {\mu} \leq 24.15+1.96\cdot \frac{5}{\sqrt{100}} \right) = 0.95$$


In [2]:
from sympy import *
1.96 * 5 / sqrt(100)

0.980000000000000

In [3]:
24.15 - 0.98

23.169999999999998

In [4]:
24.15 + 0.98

25.13

Точный доверительный интервал для математического ожидания будет:

$$P \left(23.17 \leq {\mu} \leq 25.13 \right) = 0.95$$